src/dlgprefreplaygaindlg.ui

```diff
@@ -21,22 +21,22 @@
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
-       <widget class="QCheckBox" name="EnableAnalyser">
+       <widget class="QCheckBox" name="EnableGain">
         <property name="toolTip">
-         <string>Calculate ReplayGain normalization for tracks which are missing ReplayGain metadata.</string>
+         <string>Apply loudness normalization to loaded tracks.</string>
         </property>
         <property name="text">
-         <string>Enable ReplayGain Analysis</string>
+         <string>Apply ReplayGain</string>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="QCheckBox" name="EnableGain">
+       <widget class="QCheckBox" name="EnableAnalyser">
         <property name="toolTip">
-         <string>Apply loudness normalization to loaded tracks.</string>
+         <string>Calculate ReplayGain normalization for tracks which are missing ReplayGain metadata.</string>
         </property>
         <property name="text">
-         <string>Apply ReplayGain</string>
+         <string>Enable ReplayGain 1.0 Analysis</string>
```

Comment:
Since RG2 is approaching, we should state somewhere that Mixxx uses RG1.
If RG1 is not sufficient, the user is free to use third party tool to write RG2 to the files meta data.